rooms.json

```diff
@@ -1,51 +1,163 @@
 [
   {
     "room_num": 1,
-    "room_name": "Home",
-    "desc": ["This is your home.",
-            "I got to rest up before adventuring. Crazy stuff out there."],
+    "room_name": ["Shuttle Interior (Crashed Reactor Active)","Shuttle Interior (Crashed Reactor Shutdown)"],
+    "desc": ["You stand on the slanted floor; the impact absorbing design has crumpled the walls and floor slightly to save you in the crash landing. Warning lights flash on and off, washing the area in red light intermittently. The storage lockers have broken open violently and their contents lie scattered throughout the room. The impact foam and your crash harness saved you from the crushing forces that deformed the control panel beyond usability dangle from the remains of the pilot’s seat. The speakers that are still functioning warn of imminent reactor failure.", "The warning lights have shut off with the shutdown of the reactor and the loss of power. Emergency lights running from the battery provide enough light to see and lead to the airlock hatch. Casual inspection of the seal shows that the automatic mechanisms are smashed and if you want out you will have to pop the locks manually to free yourself from the wreckage of your shuttle." ]
```

Comment:
Remove brackets to change it to a normal String. In the console, this will also print out one big line. add multiple "\n" to ensure better readability. Change String[] desc to String desc in the Room.java.

Reply:
I added the [] because there are 2 room names and 2 descriptions for rooms 1 and 15.